src/clinical/donor-repo.ts

```diff
@@ -19,29 +19,15 @@
 
 import { DeepReadonly } from 'deep-freeze';
 import mongoose, { PaginateModel } from 'mongoose';
-import mongoosePaginate from 'mongoose-paginate-v2';
```

Comment:
Changes in `donor-repo` are just reorganization; code moved to `schemas.ts` to separate concerns and reduce file size